lib/rubocop/cop/mixin/range_help.rb

```diff
@@ -101,17 +105,27 @@ def directions(side)
         end
       end
 
-      def final_pos(src, pos, increment, newlines, whitespace)
+      # rubocop:disable Metrics/ParameterLists
+      def final_pos(src, pos, increment, continuations, newlines, whitespace)
```

Comment:
I imagine this method is not meant to be used directly? I really need to finish #8278 ...

Reply:
Indeed, there's a [comment](https://github.com/rubocop-hq/rubocop/pull/8448/files#diff-fe7a38dd2048339cac7b172ed7cf2668R85) above that says these helpers are not supposed to be used directly. I agree it would be better to mark them individually as `@api private`.